_posts/2021-12-17-sprint-report-3.md

```diff
@@ -0,0 +1,73 @@
+---
+author: Tyler Weaver
+comments: false
+date: 2021-12-17 17:00:00+07:00
+layout: post
+slug: sprint-report-3
+title: MoveIt Sprint Report
+description: Nov 26 - Dec 16
+media_type: image
+media_link: /assets/images/blog_posts/2021_11_24_sprint_update.png
+categories:
+- MoveIt
+- ROS
+---
+
+## Nov 26 - Dec 16
+
+We have just finished our third MoveIt sprint using ZenHub for planning and would like to share with you what the PickNik team is doing to improve MoveIt.  From a high level the major efforts of each of the team members did:
+
+- [Abishalini](https://github.com/Abishalini) continues to lead the work of forward porting from MoveIt on ROS 1.  In this sprint, she merged a particularly large forward port containing some nuanced cmake changes.
+- [AndyZe](https://github.com/AndyZe) finished getting the [Hybrid Planner](https://moveit.picknik.ai/main/doc/hybrid_planning/hybrid_planning_tutorial.html) merged.  He also landed a tutorial for it to make it easier for users.
+- [DLu](https://github.com/DLu) landed his change to improve robot_description handling and opened a PR that refactors and ports the framework for MoveIt Setup Assistant.
+- [henningkayser](https://github.com/henningkayser) finished his port of the [Pilz Industrial Motion Planner](https://moveit.picknik.ai/main/doc/pilz_industrial_motion_planner/pilz_industrial_motion_planner.html?highlight=pilz) and got that merged.
+- [JafarAbdi](https://github.com/JafarAbdi) released rviz_visual_tools, moveit_visual_tools, and moveit_msgs into various ROS 2 distros.
```

Comment:
```suggestion
- [JafarAbdi](https://github.com/JafarAbdi) released [rviz_visual_tools](https://index.ros.org/p/rviz_visual_tools/), [moveit_visual_tools](https://index.ros.org/p/moveit_visual_tools/), and [moveit_msgs](https://index.ros.org/p/moveit_msgs/) into various ROS 2 distros.
```

Are these the links we should use or should I link to the repo or something else?